library/src/main/java/com/novoda/downloadmanager/lib/Downloads.java

```diff
@@ -26,8 +26,50 @@
  *
  * @pending
  */
-final class Downloads {
-    private Downloads() {
+public class Downloads {
+
+    private final String authority;
+
+    /**
+     * The content URI for accessing publicly accessible downloads (i.e., it requires no
+     * permissions to access this downloaded file)
+     */
+    public Uri getPublicityAccessibleDownloadsUri() {
```

Comment:
Yes but because extracting the class into a new file will cause a lot of pain when https://github.com/novoda/download-manager/pull/63 is merged I prefer to delay it (description :p)